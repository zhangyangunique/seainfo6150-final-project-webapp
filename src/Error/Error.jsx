import React from 'react'
import styles from "./Error.module.css";

const Error = () => {
	return (
		<div className={styles.errorContainer}>
			<h1>PAGE NOT FOUND</h1>
			<h1>404</h1>
			<p>Out of nothing, something.</p>
			<a href="/">Home</a>
		</div>
	)
}

export default Error
