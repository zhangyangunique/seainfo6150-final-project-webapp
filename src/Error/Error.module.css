.errorContainer {
	font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
	font-size: 0.8rem;
	text-align: center;
	margin-top: 60px;
}

@media screen and (min-width: 525px) {
	.errorContainer {
		font-size: 1rem;
		margin-top: 90px;
	}
}

@media screen and (min-width: 800px) {
	.errorContainer {
		font-size: 1.5rem;
		margin-top: 120px;
	}
}

@media screen and (min-width: 1150px) {
	.errorContainer {
		font-size: 2rem;
		margin-top: 150px;
	}
}