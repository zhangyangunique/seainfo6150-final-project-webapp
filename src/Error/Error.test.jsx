import React from "react";
import { render } from "@testing-library/react";
import Error from "./Error";

describe("Error tests", () => {
    it("renders correctly", () => {
        const { container } = render(<Error />);
        expect(container).toMatchSnapshot();
    });
});