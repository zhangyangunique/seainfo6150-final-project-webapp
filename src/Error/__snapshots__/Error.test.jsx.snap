// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests renders correctly 1`] = `
<div>
  <div
    class="errorContainer"
  >
    <h1>
      PAGE NOT FOUND
    </h1>
    <h1>
      404
    </h1>
    <p>
      Out of nothing, something.
    </p>
    <a
      href="/"
    >
      Home
    </a>
  </div>
</div>
`;
